### PR TITLE
Reference todo lists with slugs

### PR DESCRIPTION
**Pull request by junusheva**
### Description
- Add a nullable ‘slug’ field to the Todo model
- Update exporting and importing csv with todos checklist logic
- Update the logic of creating todos from templates

### CODE REVIEW

File: orchestra/tests/helpers/fixtures.py [hunk 1]
```diff
@@ -39,49 +39,54 @@
...
 
-TODO_TEMPLATE_GOOD_CSV_TEXT = """Remove if,Skip if
-[],[],the root
-[],[],,todo parent 1
-[],"[{""prop"": {""value"": true, ""operator"": ""==""}}]",,,todo child 1-1
-"[{""prop"": {""value"": true, ""operator"": ""==""}}]",[],,todo parent 2
-[],[],,,todo child 2-1
-[],[],,,,todo child 2-1-1
-[],[],,,todo child 2-2
+TODO_TEMPLATE_GOOD_CSV_TEXT = """Slug,Remove if,Skip if
```
Comment (marcua) [inline orchestra/tests/helpers/fixtures.py:42]: Can you test the import and export of one of these rows with a slug, while you leave the rest empty?

File: orchestra/models/core/models.py [hunk 2]
```diff
@@ -753,6 +753,10 @@ class Todo(TodoMixin, BaseModel):
         activity_log (str)
             A JSON blob that records the user actions
             with this todo
+        slug (str)
+            A unique identifier for each todo list item.
+            It is used to reference and retrieve specific
```
Comment (paopow) [inline orchestra/models/core/models.py]: It is used to refer and retrieve specific...

Comment (junusheva) [reply orchestra/models/core/models.py]: to-do items. Line below. Or do you mean something else?

Comment (paopow) [reply orchestra/models/core/models.py]: It is used to reference -> It is used to refer

Comment (junusheva) [reply orchestra/models/core/models.py]: oh sorry, didn't notice

**Review by paopow — APPROVED** [review]

**Pull request merged**